docs/chart_tests.md

```diff
@@ -34,19 +34,33 @@ mariadb/
   templates/
   templates/mariadb-tests.yaml
```

Comment:
We'll probably want to change this chart structure also to match the example below.